src/awkward/_nplikes/numpylike.py

```diff
@@ -444,10 +444,6 @@ def repeat(
     ) -> ArrayLike:
         ...
 
-    @abstractmethod
-    def tile(self, x: ArrayLike, reps: int) -> ArrayLike:
-        ...
```

Comment:
Tile is a more general function than `broadcast_to`, and thus far we don't need that  level of power. I'm removing it so that we have less work to do!